Makefile

```diff
@@ -67,17 +67,29 @@ ifeq ("$(TARGETOS)", "android")
 		TOOLCHAIN = "aarch64-linux-android-4.9"
 		GCCBIN = "aarch64-linux-android-g++"
 	else ifeq ("$(TARGETARCH)", "arm")
+		# For ARM32, gcc cross-compiler in Android toolchain
+		# gives an error.
+		# Using clang cross-compiler in android NDK toolchain works.
+		CC = $(NDKARM)/bin/clang++
```

Comment:
Can we do the same as we do other arches?
Namely, require setting only $(NDK) and locate the non-standalone toolchain there.
It seems that it's doable by setting TOOLCHAIN=llvm GCCBIN=clang and then adding some CLAGS.